docs/concepts/flows.md

```diff
@@ -204,158 +181,81 @@ if __name__ == "__main__":
 
-## Visualizing flow structure
+By effectively managing flow runs, you can ensure the smooth execution and maintainability of your workflows.
 
-You can get a quick sense of the structure of your flow using the `.visualize()` method on your flow. Calling this method will attempt to produce a schematic diagram of your flow and tasks without actually running your flow code.
+### Flow run states
 
-!!! warning "Functions and code not inside of flows or tasks will still be run when calling `.visualize()`. This may have unintended consequences. Place your code into tasks to avoid unintended execution."
+Prefect tracks the state of each flow run, allowing you to monitor its progress and identify any issues. Here are the typical flow run states:
 
-!!! note "To use the `visualize()` method, Graphviz must be installed and on your PATH. Please install Graphviz from [http://www.graphviz.org/download/](http://www.graphviz.org/download/). And note: just installing the `graphviz` python package is not sufficient."
+* **Scheduled:** The flow run is waiting for execution based on a schedule.
+* **Running:** The flow and its tasks are actively executing.
+* **Failed:** The flow run encountered an error and did not complete successfully.
+* **Cancelled:** The flow run was manually stopped before completion.
+* **Succeeded:** The flow run finished execution without errors.
 
-```python
-from prefect import flow, task
+## Create modular workflows using subflows
 
-@task(name="Print Hello")
-def print_hello(name):
-    msg = f"Hello {name}!"
-    print(msg)
-    return msg
+Subflows allow you to create modular units of work within your Prefect workflows. These subflows can contain multiple tasks, offering several advantages over using individual [tasks](/concepts/tasks/):
 
-@task(name="Print Hello Again")
-def print_hello_again(name):
-    msg = f"Hello {name}!"
-    print(msg)
-    return msg
+* **Enhanced observability**: Subflows have first-class observability within Prefect, providing clear visibility into their status within the UI and Prefect Cloud.  You can easily monitor subflow progress in the Flow Runs dashboard, eliminating the need to delve into individual tasks within a specific flow run. See [states](/concepts/states/) for examples utilizing task state within flows.
 
-@flow(name="Hello Flow")
-def hello_world(name="world"):
-    message = print_hello(name)
-    message2 = print_hello_again(message)
+* **Conditional execution**: Subflows enable you to group tasks that only run under specific conditions. This simplifies conditional logic by allowing you to conditionally run the entire subflow instead of managing individual tasks.
 
-if __name__ == "__main__":
-    hello_world.visualize()
-```
+* **Parametrization**: Prefect's parameterization capabilities extend to subflows. This allows you to reuse the same group of tasks for different purposes by simply passing different parameters when invoking the subflow.
```

Comment:
Throughout, sometimes the colon is included in the bolded text and sometimes not. Either is fine as long as its consistent.